File: Programming Basics with JS/T02 - First Steps in Coding/lecture/09. Yard Greening.js
function yardGreening(input) {
    let sqM = (input[0]);
    let pricePerSqM = 7.61;
    let discountFromPrice = 18 / 100;
    let totalPrice = sqM * pricePerSqM;
    let totalDiscount = totalPrice * discountFromPrice;
    let finalPrice = totalPrice - totalDiscount;

    console.log("The final price is: " + finalPrice + " lv.");
    console.log("The discount is: " + totalDiscount + " lv.");
}
yardGreening([550]);

File: Programming Basics with Go/T02 - First Steps in Coding/lecture/09. Yard Greening.go
package main

import "fmt"

func main() {
	var sqm float32
	fmt.Scanln(&sqm)

	var totalPrice float32 = sqm * 7.61
	var discount float32 = totalPrice * 0.18
	var leftSum float32 = totalPrice - discount

	fmt.Printf("The final price is: %f lv.\n", leftSum)
	fmt.Printf("The discount is: %f lv.", discount)
}